File: app/src/main/java/com/rR/apksapp/utils/AppConstants.java
package com.rR.apksapp.utils;

import android.os.Environment;

import java.text.SimpleDateFormat;
import java.util.Date;

public class AppConstants {
    public static final int GRID_COLUMNS = 3;
    public static final int REQUEST_CODE_STORAGE_PERMISSION = 1234;
    public static final String FOLDER_NAME = "AppsAPK";
    public static final String FOLDER_PATH = Environment.getExternalStorageDirectory().
            getAbsolutePath();
    public static String CURRENT_INSIDE_FOLDER_NAME = getTodayDateWithCurrentTime();
    private static final String SIMPLE_DATE_FORMAT = "dd-MM-yyyy";

    private static String getTodayDateWithCurrentTime() {
        SimpleDateFormat simpleDateFormat = new SimpleDateFormat(SIMPLE_DATE_FORMAT);
        return simpleDateFormat.format(new Date(System.currentTimeMillis()));
    }

}

File: app/src/main/java/android/rr/apksapp/utils/AppConstants.kt
package android.rr.apksapp.utils

import android.os.Environment
import java.text.SimpleDateFormat
import java.util.*

object AppConstants {
    const val GRID_COLUMNS = 3
    const val REQUEST_CODE_STORAGE_PERMISSION = 1234
    val FOLDER_NAME: String? = "AppsAPK"
    val FOLDER_PATH = Environment.getExternalStorageDirectory().absolutePath
    var CURRENT_INSIDE_FOLDER_NAME = getTodayDateWithCurrentTime()
    private val SIMPLE_DATE_FORMAT: String? = "dd-MM-yyyy"
    private fun getTodayDateWithCurrentTime(): String? {
        val simpleDateFormat = SimpleDateFormat(SIMPLE_DATE_FORMAT)
        return simpleDateFormat.format(Date(System.currentTimeMillis()))
    }
}